### PR TITLE
don't fling held items if someone else is watching

### PR DESCRIPTION
**Pull request by sethalves**
- fix bug that caused adjustments to equipped entities to fling the entity if someone else is watching

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- log two avatar in
- have one equip something
- have the equipper use the other hand to adjust the item a few times, while maintaining the equip
- confirm that the item doesn't jump or spin off of the equipper's hand

File: interface/src/avatar/AvatarActionHold.cpp [hunk 1]
```diff
@@ -153,32 +153,10 @@ std::shared_ptr<Avatar> AvatarActionHold::getTarget(float deltaTimeStep, glm::qu
...
-                _previousPositionalDelta = positionalDelta;
-                _previousDeltaTimeStep = deltaTimeStep;
-            }
         }
 
         rotation = palmRotation * _relativeRotation;
         position = palmPosition + rotation * _relativePosition;
-
-        // update linearVelocity based on offset via _relativePosition;
-        linearVelocity = linearVelocity + glm::cross(angularVelocity, position - palmPosition);
```
Comment (hyperlogic) [inline interface/src/avatar/AvatarActionHold.cpp]: We still need this bit.  This accounts for the additional linearVelocity at the _relativePosition due to the hand controller angularVelocity.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8f304d95b39aeda6e2e20d6a5b1dfc0eea7caba9 available at: https://gist.github.com/7114583ae63f3325ca84<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 74452bc89009753af163f6fefb2322cdb6baf45f available at: https://gist.github.com/7114583ae63f3325ca84<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.dmg

**Comment by sethalves** [comment]
let's wait on this one until Andrew's sim-ownership changes are in

**Comment by hifi-gustavo** [comment]
Build console output for commit: 95f97939997fda2c6c1c2c9e22d7b7250fddc53d available at: https://gist.github.com/7114583ae63f3325ca84<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7202.dmg

**Comment by PhilipRosedale** [comment]
Tested with Seth in demo, works adjusting with both hands, equipping.

**Pull request merged**